### ISSUE TITLE
Linux Mint: OS-level evdev key remap ignored, only in newer vscode versions

### ISSUE THREAD

**Issue by codingthat** [Microsoft/vscode#24690]
I finally caved to vscode's nagging and upgraded it (from a version I had downloaded in February, IIRC, but I'm not sure the version number) only to find that my custom keybinding suddenly broke.  Since the key in question is the "end" key, this basically makes vscode unusable for me, which is really unfortunate, because I was really liking it so far.  I'm on Linux Mint 18.1 x64:

- VSCode Version: Code 1.11.1 (d9484d12b38879b7f4cdd1150efeb2fd2c1fbf39, 2017-04-06T13:55:56.395Z)
- OS Version: Linux x64 4.4.0-53-generic
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|vscode-database|bajdzis|1.2.0|
|vshaxe|nadako|1.5.0|;

I have an external USB Mac keyboard plugged into my acer laptop.  Because the Mac keyboard has no End key, I remapped the keypad Enter to cover this functionality; it's conveniently placed, without being easy to make a typo on, and takes up a redundant key.

I did this with the following line:

```
	<END> = 104; // instead of 115, so my external Mac keyboard has an "end" button (the numpad Enter)
```

in `/usr/share/X11/xkb/keycodes/evdev`.

Everywhere else in the OS this still works, and it worked in vscode too, before the upgrade just now.

After the upgrade, I tried to fix things using vscode's "Keyboard Shortcuts" preflet, but when I try to map cursorEnd to the numpad enter key, it gets input as "enter".  After that, it works, but then my regular Enter key (above Shift) now also functions as cursorEnd, so I have no way of inserting linebreaks.

I hope the behaviour is reverted in future versions.

Thankfully I was able to downgrade by installing from here http://packages.microsoft.com/repos/vscode/pool/main/c/code/code_1.9.1-1486597190_amd64.deb and everything works again.

**Comment by rmunn** [comment]
I think you're probably hitting https://github.com/Microsoft/vscode/issues/24107, which was fixed in the 1.11.2 release just a couple of days ago. That bug report specifically mentions the `End` key as one of the key codes that weren't working for a number of people. See if 1.11.2 fixes your issue.

**Comment by codingthat** [comment]
@rmunn thanks, but actually it seems it was fixed in 1.11.1, I was just ignorant of the troubleshooting guide.  Just tried 1.11.2 and the bug was still there, but `"keyboard.dispatch": "keyCode"` fixed it immediately, so I don't think it was #24107 but #21780.  In any case, a duplicate.  My bad!
